addons/calendar/models/calendar_recurrence.py

```diff
@@ -58,13 +60,13 @@
 ]
 
 WEEKDAY_SELECTION = [
-    ('MO', 'Monday'),
-    ('TU', 'Tuesday'),
-    ('WE', 'Wednesday'),
-    ('TH', 'Thursday'),
-    ('FR', 'Friday'),
-    ('SA', 'Saturday'),
-    ('SU', 'Sunday'),
+    ('MON', 'Monday'),
+    ('TUE', 'Tuesday'),
+    ('WED', 'Wednesday'),
+    ('THU', 'Thursday'),
+    ('FRI', 'Friday'),
+    ('SAT', 'Saturday'),
+    ('SUN', 'Sunday'),
```

Comment:
Why are the day in uppercase here?

Reply:
@arj-odoo ```WEEKDAY_SELECTION``` is used as selection field options for ```weekday``` field and previously database values was in uppercase we kept same in uppercase.